src/Approvals/SFA.DAS.Approvals.Api/Controllers/CohortController.cs

```diff
@@ -64,11 +64,11 @@ public async Task<IActionResult> GetCohortDetails(long cohortId)
                     return NotFound();
                 }
 
-                return Ok((GetCohortDetailsResponse)result);
+                return Ok(result);
```

Comment:
We've lost the conversion from result -> response here. Query results are an internal concern and should not be directly exposed as an api response.

Reply:
Gotcha, fixed!